snapcraft.yaml

```diff
@@ -32,10 +32,9 @@ parts:
     source-type: git
     source-tag: 3.38.2
     plugin: meson
-    meson-parameters: [--prefix=/usr]
+    meson-parameters:
+      - --prefix=/usr
     parse-info: [usr/share/metainfo/org.gnome.Nibbles.appdata.xml]
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
-    organize:
-      snap/gnome-nibbles/current/usr: usr
+      craftctl default
+      patch -p1 < $CRAFT_PROJECT_DIR/patches/01-fix-vala-compilation-update-help-and-translations.diff
```

Comment:
Sorry based on the conversation, I thought you would go with *.diff?

Reply:
My fault, I forgot to do the commit/push. Fixed.